zipkin-server/pom.xml

```diff
@@ -199,6 +199,13 @@
       <scope>runtime</scope>
     </dependency>
 
+    <!-- Eureka service discovery -->
+    <dependency>
+      <groupId>${armeria.groupId}</groupId>
+      <artifactId>armeria-eureka</artifactId>
```

Comment:
because armeria already includes this, there's no real bloat in supporting this. It is just a small http client integration without any other deps. This is because eureka is basically a rest api.